packages/wes_adapter/amazon_genomics/wes/adapters/NextflowWESAdapter.py

```diff
@@ -136,7 +136,7 @@ def query_logs_for_job(self, job_details: JobDetailTypeDef, query: str):
             endTime=end_time or int(time.time()),
             queryString=query,
             # TODO: handle pagination? GetRunLog doesn't seem to support it...
-            limit=100,
+            limit=10_000,
```

Comment:
Is this meant to be an underscore?

Reply:
Yes. The underscore is optional but a good idea for readability of large numbers:
https://peps.python.org/pep-0515/